src/client/app/chat/chat.component.ts

```diff
@@ -108,6 +109,7 @@ export class ChatComponent implements OnInit {
   doctorDetails:any;
   showPrescriptionComponent:Boolean = false;
   digitalSignature:string;
+  displayMessageLoadingIcon:Boolean ;
```

Comment:
rename it to displayLoader or simply loader

Reply:
But there are multiple animation loaders in chat component